resources/org.ebnf

```diff
@@ -293,8 +293,10 @@ text-link-plain-path = #"[^\s()<>]+(\w|[^\s[:punct:]]/)"
    This works so far because when parsing, text-normal is tried last.
    It can result in multiple subsequent text-normal which will be
    concated in a later transform step.
+
+   Stop parsing at EOL.
  *)
-text-normal = #".[^*/_=~+\[<{^\\]*"
+text-normal = #"[^\n\r][^*/_=~+\[<{^\\\n\r]*"
```

Comment:
For self: Check again, if `.` implies all characters, also `\n\r`. If not, this can be simplified.